src/WarcraftLegacies.Source/Quests/BlackEmpire/QuestMawofGorath.cs

```diff
@@ -0,0 +1,42 @@
+﻿using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.QuestSystem;
+using WarcraftLegacies.Source.Factions;
+
+namespace WarcraftLegacies.Source.Quests.BlackEmpire
+{
+  /// <summary>
+  /// Gain an additional Forgotten one
+  /// </summary>
+  public sealed class QuestMawofGorath : QuestData
+  {
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestMawofGorath"/> class.
+    /// </summary>
+    public QuestMawofGorath(LegendaryHero zonozz) : base("Maw of Go'rath",
+      "Zon'ozz can awaken the Maw of Go'rath, a terrible Forgotten One.",
+      @"ReplaceableTextures\CommandButtons\BTNFacelessKing.blp")
+    {
+      AddObjective(new ObjectiveControlLevel(UNIT_N02Q_DRAGONBLIGHT, 10));
```

Comment:
Dragonblight seems a bit random. Is there a Forgotten One there or something? Should convey that somehow in the quest description.

Reply:
See discord